dcos/subcommand.py

```diff
@@ -141,7 +141,7 @@ def default_subcommands():
     :returns: list of all the default dcos cli subcommands
     :rtype: [str]
     """
-    return ["auth", "config", "help", "marathon",
+    return ["auth", "config", "help", "job", "marathon",
```

Comment:
I widen my search to root project level.  there is a `default_subcommands` in `subcommand.py` in dcos, but I already added `job` there.  Am I missing something?